### ISSUE TITLE
Config option to prevent stack info printing in PR description

### ISSUE THREAD

**Issue by mattd-tg** [abhinav/git-spice#289]
Different people on my team use different stacking tools (or no tool and do it manually), so we have a github action that adds the stack information to the PR description. 

It would be nice to have a config option to prevent `git-spice` from adding the stack info, because it ends up being redundant in my case.

Depends on #16
